data/builder/file.go

```diff
@@ -57,6 +58,91 @@ func BuildUnixFSFile(r io.Reader, chunker string, ls *ipld.LinkSystem) (ipld.Lin
 	}
 }
 
+// EstimateUnixFSFile estimates the byte size of the car file that would be
```

Comment:
I am curious why the word "estimate"? I read that as: the actual size may differ. Is that right? If so, would it make sense to update the godoc to elaborate on the discrepancy?

If not, I recommend avoiding this terminology.

Reply:
I think the only real discrepancy in my head at this point is that depending on your actual data, the resulting car may have de-duplicated blocks so may be smaller than expected